networking_nsxv3/plugins/ml2/drivers/nsxv3/agent/provider_nsx_policy.py

```diff
@@ -1136,7 +1136,14 @@ def outdated(self, resource_type: str, os_meta: Dict[str, dict]):
         return outdated, current
 
     def age(self, resource_type: str, os_ids: List[str]):
-        return [(resource_type, id, self.metadata(resource_type, id).age) for id in os_ids]
+        aged_resources = []
+        for id in os_ids:
+            meta_data = self.metadata(resource_type, id)
+            if meta_data:
+                aged_resources.append((resource_type, id, meta_data.age))
+            else:
+                LOG.info("Resource: %s with ID: %s has no metadata", resource_type, id)
```

Comment:
[optional] Would write `"Resource %s with ID %s has no metadata"` or `"Object %s: Missing metadata for resource %s"` - the two `:` seem weird to me.